package.json

```diff
@@ -1,5 +1,7 @@
 {
-  "name": "hardhat-project",
+  "name": "staterelayer-bot",
+  "version": "0.0.0",
+  "private": false,
```

Comment:
include these three lines: 
https://github.com/BirthdayResearch/quantum/blob/2ef63030a0aa8789a13209ab3c43f67787da4be0/packages/smartcontracts/package.json#L5

Reply:
so the packager will know what to include when building the package

Reply:
and btw, can we change the name to @waveshq/state-relayer-bot

Reply:
Ah yes, I don't see why this would be an issue.